lib/functions-api.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+function current_event_call($data) {
+  $current_event_id = IGV_get_option('_igv_site_options', '_igv_radio_evento_current');
+
+  // Check if no event is selected
+  if (empty($current_event_id)) {
```

Comment:
Yeah this is doable but i think there's no budget for this. We would need to ad end-time to events, rewrite this checker, etc
In think we should leave it as it is for now, specially because of the short budget